packages/expo-app-auth/README.md

```diff
@@ -23,10 +23,7 @@ and run `pod install`.
 
 ### iOS (no Cocoapods)
```

Comment:
I'd just remove this section from all the READMEs and have one page (that we can link to, maybe `expo/guides/Using CocoaPods.md`) that explains why we use CocoaPods. The iOS Expo client app uses CocoaPods and that is the workflow we regularly support, maintain, and test, and the best and most genuine way we can keep things working is to use the same tools for all Expo projects.

Reply:
Added https://github.com/expo/expo/pull/2919/files#diff-63279026895ab307cf0206be52ee6b09 🙂